static/scripts/upload_files.js

```diff
@@ -0,0 +1,29 @@
+﻿async function uploadTestFile() {
```

Comment:
This should be a function definition similar to this one:

https://github.com/LivingInSyn/dlptest/blob/main/static/scripts/http_test.js#L8